src/fparser/two/tests/test_fortran2003.py

```diff
@@ -2110,6 +2110,14 @@ def test_stop_stmt():  # R849
     assert isinstance(obj, tcls), repr(obj)
```

Comment:
I'm afraid we're following the "you touch it, you fix it" approach here so please add a docstring for this test.

Reply:
Ah - that approach :)